webapp/login/components/useResetPasswordFormState.js

```diff
@@ -27,7 +27,7 @@ export const useResetPasswordFormState = props => {
   const validateObj = async obj => await validate(
     obj,
     {
-      'password': [validateRequired, validatePassword, validatePasswordStrength],
+      'password': [validateRequired('Password is required'), validatePassword, validatePasswordStrength],
```

Comment:
should this be a key? something like 'validationErrors.user.passwordRequired'?

Reply:
Yes, added keys